### PR TITLE
luci-app-ssr-plus:bump to 182-5

### LINKED ISSUE #31: 能不能增加一键恢复默认选项
就是恢复到ipk刚刚安装好的时候

### LINKED ISSUE #200: 希望能支持 trojan-go 的客户端
Trojan-GO 已经从各方面可以说已经超越原版 Trojan 了，希望能支持Trojan-GO的相关配置

### LINKED ISSUE #302: 建议加入 Trojan-go 选择
Trojan-go 是 trojan 的变种，比 trojan 有更大的优势，如下

1. 支持 websocket，可以套 CDN 使用 （Trojan 不能套 cdn ）
2. 支持 udp ，并且实现了 udp 的 FullCone ，这点目前没看到有别的软件能实现 FullCone 的
3. 测试套 CF 之后，晚高峰依然速度不错，之前 v2ray 套 cdn 之后速度特别差 
4. 支持 mux ，多路复用，性能比 Trojan 更好
5. 支持 tcp redirect 和 tproxy，可以透明爬 
6.自带代理功能， 一个端口自适用  socks5 和 http proxy ，不需要用 v2ray 来做 http --> socks5，省事

目前 Trojan-go 唯一的缺点就是 客户端 没有好用的， 希望 helloworld 加上 Trojan-go 的支持，这样就无需别的客户端了

### LINKED ISSUE #317: 手动添加服务器节点时无法导入vless协议
目前仅支持ssr:// | ss:// | vmess:// | trojan://，vless://没有支持么？还是我漏编译什么包了？
ps：手动导入vless的信息后能正常使用。

### PR DESCRIPTION
**Pull request by Mattraks**
# luci-app-ssr-plus:bump to 182-5

## Important improvement
Rewrite startup code to speed up startup  
Same as global server, no need to start multiple processes  
Use unified path and configuration  

## Changes
Add Reset default settings  Fixes #31
~~Optimize the use of shadowsocks plugin~~  
Remove the sh script used in update and modify the UI usage  
Remove Easylist China from adblock and add NEO DEV HOST  
Adjust the parameters used by wget to ensure compatibility  
Add Trojan-go support  Fixes #200
Add vless share link import  Fixes #317
Optimize update return value  
V2ray_Plugin is off by default  
Improve Trojan-Go support  Fixes #302
Add port 9418 for cloning repositories using git://protocol  
Modify 'Apply' and 'Delete All Subscribe Severs' execution method

## Fixes
Fix the issue of starting twice after apply in the server node list  
Fix the problem that dnsmasq fails to start when the access control list contains wrong entries  
Replace expired gfwlist update URLs  
Fix shadowsocks startup bug  
Fix process daemon bug  
Fix vmess URL import  
Fix the subscription update failure caused by manually adding invalid nodes

### CODE REVIEW

**Comment by EnnawYang** [comment]
备份列表记得更新下（

**Review by jun129 — APPROVED** [review]

**Comment by jun129** [comment]
我只是想更新我FORK的项目。。怎么点在这了，不好意思

File: luci-app-ssr-plus/luasrc/controller/shadowsocksr.lua [hunk 1]
```diff
@@ -4,7 +4,7 @@ module("luci.controller.shadowsocksr", package.seeall)
 
 function index()
 	if not nixio.fs.access("/etc/config/shadowsocksr") then
-		return
+		call("act_reset")
```
Comment (1715173329) [inline luci-app-ssr-plus/luasrc/controller/shadowsocksr.lua:7]: I would like to keep it as-is, to avoid loop issue in some cases.

**Review by 1715173329 — CHANGES_REQUESTED** [review]

File: luci-app-ssr-plus/luasrc/controller/shadowsocksr.lua (hunk 1 above)
Comment (Mattraks) [reply luci-app-ssr-plus/luasrc/controller/shadowsocksr.lua:7]: I don't think there will be a loop problem, to determine whether the configuration file exists, and create one if it does not exist.

**Comment by BoBppy** [comment]
用 [v2ray-rules-dat](https://github.com/Loyalsoldier/v2ray-rules-dat) 结果只用了里面的gfw.list，效果与gfw/gfw.list一致（

**Pull request merged**